commands/publish/index.js

```diff
@@ -595,11 +612,16 @@ class PublishCommand extends Command {
         tracker.completeWork(1);
 
         this.execScript(pkg, "postpublish");
+        this.runPackageLifecycle(pkg, "postpublish");
       });
     };
 
-    return pFinally(runParallelBatches(this.batchedPackages, this.concurrency, mapPackage), () =>
-      tracker.finish()
+    return pFinally(
+      promise.then(() => runParallelBatches(this.batchedPackages, this.concurrency, mapPackage)),
+      () => {
+        this.runPackageLifecycle(rootPkg, "postpublish");
```

Comment:
Since this needs to be returned inside the Promise chain too, I'd prefer if the actions in this method were composed into a chain similar to the `execute()` method:

```js
let chain = this.runPrepublishScripts(rootPkg);

chain = chain.then(() =>
  pMap(this.updates, ({ pkg }) => {
    this.execScript(pkg, "prepublish");

    return this.runPrepublishScripts(pkg);
  })
);

tracker.addWork(this.packagesToPublish.length);

const mapPackage = pkg => {
  // ...
};

chain = chain.then(() => runParallelBatches(this.batchedPackages, this.concurrency, mapPackage));
chain = chain.then(() => this.runPackageLifecycle(rootPkg, "postpublish"));

return pFinally(chain, () => tracker.finish());
```